test/e2e/specs/wp-calypso-gutenberg-post-editor-spec.js

```diff
@@ -277,6 +276,12 @@ describe( `[${ host }] Calypso Gutenberg Editor: Posts (${ screenSize })`, funct
 				);
 			} );
 
+			step( 'Can see the Jetpack blocks', async function() {
```

Comment:
We need a way of skipping this step in IE11, since Jetpack blocks are currently broken there: https://github.com/Automattic/jetpack/issues/14273

Reply:
Done in f1148a8